esmvalcore/cmor/_fixes/cmip6/ciesm.py

```diff
@@ -19,9 +19,20 @@ def fix_data(self, cube):
         Returns
         -------
         iris.cube.Cube
-
         """
         metadata = cube.metadata
         cube *= 100
         cube.metadata = metadata
         return cube
+
+
+class Pr(Fix):
+    """Fixes for pr."""
+
+    def fix_metadata(self, cube):
+        """Fix metadata.
+
+        The values of v20200417 are off by a factor 1000.
+        """
+        cube.data = cube.core_data() * 1000.
```

Comment:
this is `fix_data()` man, and the args of the baseclass function are `(self, cubes)` hence the Codacy issue